SCF-Session-5/searches/SearchTest.java

```diff
@@ -0,0 +1,48 @@
+package searches;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class SearchTest
+{
+	Search searchObject = new Search();
+	
+	@Test
+	public void testPositive()
```

Comment:
Use separate test methods for each method in class and test method name should be relevant.

Reply:
done